script/test

```diff
@@ -41,40 +41,24 @@ readonly COLOR_PASS='\033[0;32m'
 readonly COLOR_FAIL='\033[0;31m'
 readonly COLOR_NONE='\033[0m'
 
-print_result() {
-    pass=$1
-
-    if ${pass}; then
-        prefix="${COLOR_PASS}PASS${COLOR_NONE}"
-    else
-        prefix="${COLOR_FAIL}FAIL${COLOR_NONE}"
-    fi
-    shift
-
-    echo -e "${prefix}" $*
-
-    ${pass}
+do_clean() {
+    [[ ! -d "${BUILD_DIR}" ]] || rm -rvf "${BUILD_DIR}"
 }
 
 do_build() {
-    [[ -d "${BUILD_DIR}" ]] || mkdir "${BUILD_DIR}"
-    (cd "${BUILD_DIR}" && cmake .. -DCMAKE_TOOLCHAIN_FILE=../cmake/arm-none-eabi.cmake -DPLATFORM_NAME="${PLATFORM_NAME}" && make -j${BUILD_JOBS})
+    mkdir "${BUILD_DIR}"
+    (cd "${BUILD_DIR}" && cmake .. $@ && make -j${BUILD_JOBS})
 }
 
-do_clean() {
-    [[ ! -d "${BUILD_DIR}" ]] || rm -rvf "${BUILD_DIR}"
-
-    do_build
+do_clean_build() {
```

Comment:
keep the original version, we can use
`./script/test clean build` to clean and build.
`./script/test build` to rebuild if there's any error.